src/sentry/static/sentry/app/components/organizations/multipleEnvironmentSelector.jsx

```diff
@@ -167,113 +187,84 @@ class MultipleEnvironmentSelector extends React.PureComponent {
     this.toggleSelected(env, e);
   };
 
+  getEnvironments() {
+    const {organization, selectedProjects} = this.props;
+    let environments = [];
+    for (const project of organization.projects) {
```

Comment:
is this the new way of doing `forEach` even on regular Arrays?

Reply:
`for ... of` let's you iterate the contents of an any iterable. Unlike for..in which gives keys.

Reply:
Yeah, just mentioning since we normally use a `organization.projects.forEach()` style loop but i don't really mind either way so 🤷‍♀️

Reply:
I can use `forEach`. I'm indifferent on the syntax and being more consistent with existing code is :+1: